e2e/benchmarks/local-benchmark/index.html

```diff
@@ -102,6 +103,9 @@ <h2>TensorFlow.js Model Benchmark</h2>
       if (params.has('run')) {
         runTimes = Number(params.get('run'));
       }
+      if (params.has('profile')) {
+        runTimes = Number(params.get('profile'));
```

Comment:
Should be profileTimes. I don't think we need to get parameters from url twice. Please clean up this part of code.